addons/account/models/account_move.py

```diff
@@ -3928,6 +3934,7 @@ def _get_chains_to_hash(self, force_hash=False, raise_if_gap=True, raise_if_no_d
 
                 if not chain_info:
                     continue
+                chain_info['journal_restrict_mode'] = journal.restrict_mode_hash_table
```

Comment:
Indeed @mairasalazar this line should be after the check for `if early_stop` on the line below

Thanks for the heads up @nmarotte :+1: